pkg/proxy/apis/proxyconfig/register.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package proxyconfig
+
+import (
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+var (
+	SchemeBuilder = runtime.NewSchemeBuilder(addKnownTypes)
+	AddToScheme   = SchemeBuilder.AddToScheme
+)
+
+// GroupName is the group name use in this package
+const GroupName = "proxyconfig"
```

Comment:
Do we want to call this `proxyconfig.k8s.io` or `kubeproxyconfig.k8s.io`? cc @deads2k @liggitt

Reply:
Waiting for folks' suggestion on the naming :)

Reply:
@deads2k @liggitt @smarterclayton @sttts bump on the naming

Reply:
definitely with the k8s.io postfix

Reply:
kubeproxy.k8s.io with the config as kind/resource?

Reply:
I like that

Reply:
I don't think we have an agreed-upon naming convention for these groups... the [Component Config doc](https://docs.google.com/document/d/1arP4T9Qkp2SovlJZ_y790sBeiWXDO6SG10pZ_UUU-Lc/edit#) doesn't seem to set any hard guidelines here. The k8s.io suffix is probably a good idea. @mikedanese?